github/code-scanning_test.go

```diff
@@ -53,6 +54,38 @@ func TestActionsService_Alert_ID(t *testing.T) {
 	}
 }
 
+func TestActionsService_UploadSarif(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc("/repos/o/r/code-scanning/sarifs", func(w http.ResponseWriter, r *http.Request) {
+		v := new(AnalysisResults)
+		json.NewDecoder(r.Body).Decode(v)
+		testMethod(t, r, "POST")
+		want := &AnalysisResults{CommitSHA: String("abc"), Ref: String("ref/head/main"), Sarif: String("abc")}
+		if !cmp.Equal(v, want) {
+			t.Errorf("Request body = %+v, want %+v", v, want)
+		}
+
+		fmt.Fprint(w, `{"commit_sha":"abc","ref":"ref/head/main","sarif":"abc"}`)
+	})
+
+	ctx := context.Background()
+	_, err := client.CodeScanning.UploadSarif(ctx, "o", "r", "abc", "ref/head/main", "abc")
+	if err != nil {
+		t.Errorf("CodeScanning.UploadSarif returned error: %v", err)
+	}
+	const methodName = "UploadSarif"
```

Comment:
Please add a blank line before this line for consistency with the rest of this repo.

Reply:
Please add a blank line before this line (79) for consistency with the rest of this repo.

Reply:
This comment has not yet been addressed.